src/Stryker.Core/Stryker.Core/Reporters/TestStatisticsReporter/JsonTest.cs

```diff
@@ -0,0 +1,13 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="JsonTest.cs" company="Stryker.CLI">
+//   Copyright 2019  Cyrille DUPUYDAUBY
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+namespace Stryker.Core.Reporters
+{
+    public class JsonTest
```

Comment:
Couldn't the TestDescription class be used for this? It seems like a one to one mapping to me.

Reply:
I keep the remark in mind, but removed from this PR